languages/swift/exercises/concept/loops/Sources/loops/loops.swift

```diff
@@ -0,0 +1,15 @@
+func hashIDs(_ ids: [Int]) -> [(Int, Int)] {
+  fatalError("hashIDs not implemented -- place your implementation here")
```

Comment:
After looking into another PR from you I find the errorMessage `Please implement the ... function` more friendly and straight forward :)